src/index.html

```diff
@@ -16,17 +19,32 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page__body">
+    <form
+      class="page__form"
+      action="#"
+      data-qa="big"
+    >
+      <input
+        class="search-bar search-bar--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try &#34;Los Angeles&#34;"
+      />
+    </form>
+
+    <form
+      class="page__form"
+      action="#"
+      data-qa="small"
+    >
+      <input
+        class="search-bar search-bar--small"
+        type="text"
+        data-qa="hover"
+        placeholder="Try &#34;Los Angeles&#34;"
```

Comment:
The class name `search-bar--small` has the same issue as the `search-bar--big` class. Class names should represent the meaning of the content, not the styles or tag names.